webui/react/.gitignore

```diff
@@ -21,3 +21,6 @@ build-storybook/
 node_modules/
 
 report.*.json
+
+# .env with ESLINT_NO_DEV_ERRORS=true allows CRA to build with linting errors
+!.env
```

Comment:
is this intentional? these are already being worked on another pr of yours.

Reply:
oops need to get rid of this